mappings/net/minecraft/util/Util.mapping

```diff
@@ -284,20 +356,63 @@ CLASS net/minecraft/class_156 net/minecraft/util/Util
 		ARG 0 futures
 			COMMENT the completable futures to combine
 	METHOD method_654 make (Ljava/lang/Object;Ljava/util/function/Consumer;)Ljava/lang/Object;
+		COMMENT {@return {@code object} initialized with {@code initializer}}
+		COMMENT
+		COMMENT <p>This is useful when initializing static fields:
+		COMMENT <pre>{@code
+		COMMENT private static final Map<String, String> MAP = Util.make(new HashMap<>(), (map) -> {
+		COMMENT     map.put("example", "hello");
+		COMMENT });
+		COMMENT }</pre>
 		ARG 0 object
 		ARG 1 initializer
 	METHOD method_655 identityHashStrategy ()Lit/unimi/dsi/fastutil/Hash$Strategy;
+		COMMENT {@return the {@link Hash.Strategy} that uses identity comparison}
+		COMMENT
+		COMMENT <p>fastutil's "reference" object types should be used instead in most cases.
 	METHOD method_656 make (Ljava/util/function/Supplier;)Ljava/lang/Object;
+		COMMENT {@return the value supplied from {@code factory}}
+		COMMENT
+		COMMENT <p>This is useful when initializing static fields:
+		COMMENT <pre>{@code
+		COMMENT private static final Map<String, String> MAP = Util.make(() -> {
+		COMMENT     Map<String, String> map = new HashMap<>();
+		COMMENT     map.put("example", "hello");
+		COMMENT     return map;
+		COMMENT });
+		COMMENT }</pre>
 		ARG 0 factory
 	METHOD method_658 getMeasuringTimeMs ()J
+		COMMENT {@return the current time in milliseconds, to be used for measuring a duration}
+		COMMENT
+		COMMENT <p>This is not the Unix epoch time, and can only be used to determine the duration
+		COMMENT between two calls of this method.
+		COMMENT
+		COMMENT @see #getMeasuringTimeNano
+		COMMENT @see #getEpochTimeMs
 	METHOD method_659 getEpochTimeMs ()J
+		COMMENT {@return the milliseconds passed since the Unix epoch}
+		COMMENT
+		COMMENT <p>This should be used to display or store the current time. {@link #getMeasuringTimeMs}
+		COMMENT should be used for determining the duration between two calls.
+		COMMENT
+		COMMENT @see #getMeasuringTimeMs
+		COMMENT @see #getMeasuringTimeNano
 	METHOD method_660 next (Ljava/lang/Iterable;Ljava/lang/Object;)Ljava/lang/Object;
+		COMMENT {@return the item succeeding {@code object} in {@code iterable}}
+		COMMENT
+		COMMENT <p>If {@code object} is {@code null} or is not in {@code iterable}, this
+		COMMENT returns the first item of the iterable.
```

Comment:
If `object` is not in `iterable`, the while loop never terminates.

![image](https://user-images.githubusercontent.com/69905075/191231003-49bcb97e-76a3-4067-b2fe-0ebe0d136cf1.png)